rpc/src/lib.rs

```diff
@@ -57,15 +57,15 @@ where
 	C::Api: BlockBuilder<Block>,
 	P: TransactionPool + 'static,
 {
-	use frame_rpc_system::{SystemApiServer, SystemRpc};
-	use pallet_transaction_payment_rpc::{TransactionPaymentApiServer, TransactionPaymentRpc};
+	use frame_rpc_system::{SystemApiServer, System};
+	use pallet_transaction_payment_rpc::{TransactionPaymentApiServer, TransactionPayment};
 
 	let mut io = RpcModule::new(());
 	let FullDeps { client, pool, deny_unsafe } = deps;
 
-	io.merge(SystemRpc::new(client.clone(), pool.clone(), deny_unsafe).into_rpc())?;
+	io.merge(System::new(client.clone(), pool.clone(), deny_unsafe).into_rpc())?;
 
-	io.merge(TransactionPaymentRpc::new(client.clone()).into_rpc())?;
+	io.merge(TransactionPayment::new(client.clone()).into_rpc())?;
```

Comment:
This needs a test of 'transaction' after upgrade, and also test for 'explorer' to check existing data from previous versions.